src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,55 @@
+package duke.task;
+
+/**
+ * Represents a task.
+ */
+public class Task {
+    protected String description;
+    /** Whether the task has been completed */
+    protected boolean isDone;
+
+    /**
+     * Creates a task with the specified description.
+     * By default, task is not yet completed.
+     *
+     * @param description Description of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns the status icon for the task (based on whether it is completed).
+     * The icon is a string of length 1 ("X" if completed, or " " if not).
+     *
+     * @return Status icon for the task.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
```

Comment:
Suggest to keep consistent the reference of attributes, between this.isDone or isDone.

Reply:
That's true, I've changed all to `isDone`. Thanks!